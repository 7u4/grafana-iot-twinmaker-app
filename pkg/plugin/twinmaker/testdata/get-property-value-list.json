{
    "PropertyValues": {
        "bounds": {
            "PropertyReference": {
                "ComponentName": "Space",
                "EntityId": "Factory_aa3d7d8b-6b94-44fe-ab02-6936bfcdade6",
                "ExternalIdProperty": null,
                "PropertyName": "bounds"
            },
            "PropertyValue": {
                "BooleanValue": null,
                "DoubleValue": null,
                "Expression": null,
                "IntegerValue": null,
                "ListValue": [
                    {
                        "BooleanValue": null,
                        "DoubleValue": 105.09,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": null
                    },
                    {
                        "BooleanValue": null,
                        "DoubleValue": 9.5,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": null
                    },
                    {
                        "BooleanValue": null,
                        "DoubleValue": 175.96,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": null
                    }
                ],
                "LongValue": null,
                "MapValue": null,
                "RelationshipValue": null,
                "StringValue": null
            }
        }
    }
}