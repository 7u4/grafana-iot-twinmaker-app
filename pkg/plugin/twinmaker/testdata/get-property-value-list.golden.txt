ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: bounds
Dimensions: 1 Fields by 3 Rows
+------------------+
| Name: Value      |
| Labels:          |
| Type: []*float64 |
+------------------+
| 105.09           |
| 9.5              |
| 175.96           |
+------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////QAEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAIQAAAADAAAAUAAAACgAAAAEAAAAVP///wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAB0////CAAAABAAAAAGAAAAYm91bmRzAAAEAAAAbmFtZQAAAACY////CAAAABgAAAANAAAAeyJjdXN0b20iOnt9fQAAAAQAAABtZXRhAAAAAAEAAAAYAAAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAADAUwAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABQAAAFZhbHVlAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAgAFAAAAVmFsdWUAAAAAAAAA/////4gAAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAAAYAAAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAAA4AAAAAwAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABgAAAAAAAAAAAAAAAEAAAADAAAAAAAAAAAAAAAAAAAA9ihcj8JFWkAAAAAAAAAjQB+F61G4/mVAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADgAAAAAAAMAAQAAAFABAAAAAAAAkAAAAAAAAAAYAAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAACEAAAAAwAAAFAAAAAoAAAABAAAAFT///8IAAAADAAAAAAAAAAAAAAABQAAAHJlZklkAAAAdP///wgAAAAQAAAABgAAAGJvdW5kcwAABAAAAG5hbWUAAAAAmP///wgAAAAYAAAADQAAAHsiY3VzdG9tIjp7fX0AAAAEAAAAbWV0YQAAAAABAAAAGAAAAAAAEgAYABQAEwASAAwAAAAIAAQAEgAAABQAAABEAAAATAAAAAAAAwFMAAAAAQAAAAwAAAAIAAwACAAEAAgAAAAIAAAAEAAAAAUAAABWYWx1ZQAAAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAIABQAAAFZhbHVlAAAAaAEAAEFSUk9XMQ==
